web-portal/components/SubmissionForm.vue

```diff
@@ -588,49 +565,33 @@
         return this.$store.getters.GetActiveVenues
       },
 
-      eventIsPostponed: boolToTag('condition:postponed'),
-      eventIsCancelled: boolToTag('condition:cancelled'),
-      eventIsSoldOut: boolToTag('condition:sold-out'),
+      eventIsPostponed: boolToCondition('postponed'),
+      eventIsCancelled: boolToCondition('cancelled'),
+      eventIsSoldOut: boolToCondition('sold-out'),
 
-      eventMode: radioToTag('mode', /^mode:([\w-]+)$/),
-      eventCategory: radioToTag('category', /^category:([\w-]+)(:(.+))?$/),
+      eventCategory: {
+        get () {
+          return (this.calendar_event && this.calendar_event.category) ? this.calendar_event.category.split(':')[0] : ''
+        },
+        set (newValue) {
+          this.calendar_event.category = newValue
+        }
+      },
       eventCategoryOther: {
         get () {
-          const tag = this.calendar_event && this.calendar_event.tags.find(tag => /^category:other/.test(tag))
+          const tag = (this.calendar_event && /^other/.test(this.calendar_event.category)) ? this.calendar_event.category : false
```

Comment:
This definitely works as-is, but I wonder if the regex would be safer / more future-proof if it looked for `/^other:/`.

Reply:
aaahhh interesting! i like it, except that there's one valid case that it doesn't work for: plain old `other` with no further specification. while we do have the prompt for people to tell us what they mean by `other`, they're not **required** to add more info there. so as things currently stand, that's a valid category label, and when i test this out locally it works as i expect it to. thoughts?

(in case you wondered what's currently in the db, like i did) we currently have four events in the db that use the `other` category option. only one of them has the underspecified `category:other` tag, and it's a duplicate (which reminds me that we also need to do [this little bit of database cleanup](https://1nfinite1ndustries.slack.com/archives/C1DUR2VNW/p1685894279849259)).
```
infinite-api=# select id, tag from (select id, unnest(tags) as tag from events) asdf where tag like 'category:other%';
                  id                  |                                     tag
--------------------------------------+------------------------------------------------------------------------------
 7e875f02-22e0-4a2e-8823-fb345b87f878 | category:other:Ticket lottery
 7e875f02-22e0-4a2e-8823-fb345b87f878 | category:other
 6d40e65c-6016-4b42-beb7-7371662e00ff | category:other:Watch-on-demand
 f7102989-3c67-4943-9123-b71acf26037c | category:other:Recurring open mic the first and third Tuesday of every month
 c1d2c113-1d43-4487-a007-c17045e7916b | category:other:Open auditions
(5 rows)
```

Reply:
Ah! That's probably why it didn't have the `:` initially, and I just forgot about it. When I wrote this comment I was thinking it would be `category:other:` if no description was provided.

Reply:
Might also be related to #409.